plugins/codescene/README.md

```diff
@@ -60,3 +60,34 @@ proxy:
 codescene:
   baseUrl: https://codescene.my-company.net # replace with your own URL
 ```
+
+5. Adding the codescene plugin to Entity page:
+
+```yaml
```

Comment:
Maybe use the `diff` syntax here? Example:
https://github.com/backstage/backstage/blob/192b4f9de1b6d53cd1e5a8f7cd307ce8e5ac9251/plugins/catalog/README.md?plain=1#L31-L50

Reply:
Its not a diff and its a new entity way of showing the dashboard completely different from other existing pages